### ISSUE TITLE
[DataCap Application] XUNKU

### ISSUE THREAD

**Issue by javagarora736** [filecoin-project/filecoin-plus-large-datasets#216]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: XUNKU
- Website / Social Media: www.xunkukeji.com
- Total amount of DataCap being requested (between 500 TiB and 5 PiB): 5 PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB): 50 TiB
- On-chain address for first allocation: f1dq3eknotlnhyh5e46ogujd53eoevcxejufsmh2y

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
Founded in 2011, XUNKU has been committed to the development of aPaas platform with low code/no code, and developed industry Saas application products in collaboration with the head industry. We have saved tens of thousands of hours for customers and thousands of people's continuous labor costs. Xunku is committed to providing efficient and intelligent integrated development system platform for enterprise management. It is one of the fast growing software service providers in China. More than thousands of users choose Xunku technology for digital upgrade. Through a series of business chain coverage, highly integrated, easy-to-use business applications, to help enterprises digital reform, reduce costs and increase efficiency, release the growth potential of the company.
The unique value of xunku is very easy to use and fully integrated development.
```

What is the primary source of funding for this project?
```
Shareholder investment, business income
```

What other projects/ecosystem stakeholders is this project associated with?
```
More than 100 enterprises have chosen Xunku aPaaS enabling enterprise digitization.
Such as: China Post, China Telecom, China Unionpay...
```

## Use-case details

Describe the data being stored onto Filecoin
```
We help customers build a lot of public mini programs, platforms, websites and so on. According to the agreement, this part of data is publicly available, and we want to store them. Besides, we have a lot of platforms to store.
```

Where was the data in this dataset sourced from?
```
From our own platform.
From our customers, but all data comes from public pages.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
https://oa.xunkukeji.com/#/5010376_1000047/1000525
Additional, for mini programs, you can search: 融E家旗舰店，浙里汇，优图云学院...
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
Yes, all public.
```

What is the expected retrieval frequency for this data?
```
From time to time.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
If it's due, we want to renew it
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
Aisa.
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
Online transfer.
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We would like to use filfox.info to check the miner's situation. And we would like to find the miners in Slack, github.
```

How will you be distributing deals across storage providers?
```
We would like to choose more than 3 miners, and no more than 33% for each miner.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Yes, enough.
```

**Comment by galen-mcandrew** [comment]
@javagarora736 Please provide a sample dataset and official contact email

**Comment by javagarora736** [comment]
Sure, our official contact email: xunku@xunkukeji.com

**Comment by javagarora736** [comment]
And more sample data, the website we made: https://m.ule.com

**Comment by javagarora736** [comment]
@galen-mcandrew

**Comment by javagarora736** [comment]
@galen-mcandrew

**Comment by javagarora736** [comment]
@galen-mcandrew Hi, Galen, any update?

**Comment by javagarora736** [comment]
@galen-mcandrew

**Comment by javagarora736** [comment]
@galen-mcandrew Hi, Galen, any update?

**Comment by javagarora736** [comment]
@galen-mcandrew

**Comment by javagarora736** [comment]
@galen-mcandrew Hi, Galen. Anything else should I offer?

**Comment by javagarora736** [comment]
@galen-mcandrew

**Comment by javagarora736** [comment]
@galen-mcandrew Hi

**Comment by NDLabs** [comment]
@galen-mcandrew

**Comment by Sunnyiscoming** [comment]
@javagarora736 Hi. It seems that you want to store data related with your partner cooperated before.Have you established cooperation with prospective storage providers?

**Comment by beatlesgogo** [comment]
Yes, the data we want to store not only related with our partner, also a lot of data from platforms we have to store.

**Comment by javagarora736** [comment]
@Sunnyiscoming Hello！Sunnyiscoming,Yes, the data we want to store not only related with our partner, also a lot of data from platforms we have to store.   Until now, we have about more than 5PiB data needs to be store. We have already prepared data for the first round. That's why i asked so many times. 
Also, we have communicated with several SPs already.Thank you！

**Comment by dkkapur** [comment]
@javagarora736 great to hear that the data is prepared and ready to go! How many SPs are you going to work with and how did you find them? 

@galen-mcandrew let's get this kicked off ASAP and notaries can help with additional due diligence as needed.

**Comment by javagarora736** [comment]
Hello, Deep @dkkapur 
We found the SPs on the official platform of miner announced by FIL. 
Planing to have 10 SPs at beginning. 
Thanks!

**Comment by galen-mcandrew** [comment]
## Multisig Notary requested
#### Total DataCap requested 
> 5PiB
#### Expected weekly DataCap usage rate 
> 50TiB

**Comment by DarnellWashington** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecosa6wfti32ygydmm36wqjcncy6el54mkvwpadql54v5tbgwra5e 
#### Address 
> f1dq3eknotlnhyh5e46ogujd53eoevcxejufsmh2y
#### Datacap Allocated
> 25.00TiB
#### Signer Address
> f1bjdcjxha3ldcstw5zmvkyu3r2p5x2bsm745kgsi
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecosa6wfti32ygydmm36wqjcncy6el54mkvwpadql54v5tbgwra5e

**Comment by xunku1122** [comment]
Hello, @dkkapur  @galen-mcandrew  @Sunnyiscoming  our github account of xunku has been lost, we will replace this account for future contact。
